ballerina/types.bal

```diff
@@ -81,42 +128,42 @@ public type DcObject record {
 # Represents the status of the operation
 public enum Status {
     SUCCESS,
-    OPERATIONS_ERROR,
-    PROTOCOL_ERROR,
-    TIME_LIMIT_EXCEEDED,
-    SIZE_LIMIT_EXCEEDED,
-    COMPARE_FALSE,
-    COMPARE_TRUE,
-    AUTH_METHOD_NOT_SUPPORTED,
-    STRONGER_AUTH_REQUIRED,
+    OPERATIONS_ERROR = "OPERATIONS ERROR",
```

Comment:
Is there an additional advantage when specifying an expression like this. I think it would be better if we can have proper doc-comments for each enum value describing the usecase.

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-ldap/pull/46/commits/9fceddfb9533db9c325319270375be428241d36d